README.md

```diff
@@ -85,6 +85,7 @@ This is a list of all variables that can be used:
 | `DEPARTMENT`               | no       | SSL Certificate registration department. Used when `CERTIFICATE_MODE` is `AUTO_GENERATE` or `SELF_SIGNED`.                                                                                                                                                                                | `Department of Information Security`                                                 |
 | `CHT_COMPOSE_PROJECT_NAME` | no       | docker-compose project name to use for CHT-Core. Defaults to `cht`                                                                                                                                                                                                                        | `cht-dev-4-alpha`                                                                    |
 | `CHT_NETWORK`              | no       | docker network to use for cht-core and cht-upgrade-service. Defaults to `cht-net`                                                                                                                                                                                                         | `cht-dev-4-alpha-net`                                                                |
+| `CHT_BACKUP_COMPOSE_FILES` | no       | Whether to backup yml files before overwriting. Can be `true` or `false` (default `true`).                                                                                                                                                                                               | `false`                                                                              |
```

Comment:
Just for learning, is this made an option in the env variables mainly because of the unit tests? 
It doesn't look that this is consuming megas of space, and the process doesn't look heavy.

Reply:
We could change the tests so they're not dependent on this variable. I added it as this PR adds new requirements to the service (writable file system, disk space) so I wanted an option to disable it, but I'm happy to remove the option as it might be unnecessary complexity